libenkf/src/enkf_config_node.c

```diff
@@ -603,6 +607,43 @@ const char * enkf_config_node_get_enkf_infile( const enkf_config_node_type * con
   return path_fmt_get_fmt( config_node->enkf_infile_fmt );
 }
 
+
+const char * enkf_config_node_get_init_file(enkf_config_node_type * config_node, const enkf_main_type * enkf_main ) {
+  if (config_node->init_file_abs_path != NULL)
+    return config_node->init_file_abs_path;
+
+  model_config_type * model_config = enkf_main_get_model_config(enkf_main);
+  char * runpath                   = NULL;
+  char * path_to_init_file         = NULL;
+  char * abs_path_to_init_file     = NULL;
+  bool forward_init                = enkf_config_node_use_forward_init(config_node);
+
+  if (forward_init) {
+    path_fmt_type * runpath_fmt = model_config_get_runpath_fmt(model_config);
+    runpath = path_fmt_alloc_path(runpath_fmt , false , 0, 0);  /* Replace first %d with iens, if a second %d replace with iter */
+    path_to_init_file = enkf_config_node_alloc_initfile(config_node, runpath, 0);
+  } else
+    path_to_init_file = enkf_config_node_alloc_initfile(config_node, NULL, 0);
+
+  if (path_to_init_file)
```

Comment:
The existence check can be directly on the `path_to_init_file` variable, not necessary to first create the absolute variety.